src/schema/v1/fields/money.ts

```diff
@@ -105,6 +109,37 @@ const money = ({ name, resolve }) => ({
   }),
 })
 
+export const Price = new GraphQLObjectType<any, ResolverContext>({
+  name: "Price",
+  fields: {
+    minor: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: "An amount of money expressed in minor units (like cents).",
+      resolve: ({ cents }) => cents,
+    },
+    currencyCode: {
+      type: new GraphQLNonNull(GraphQLString),
+      description:
+        "The ISO-4217 alphabetic currency code, as per https://en.wikipedia.org/wiki/ISO_4217",
+      resolve: ({ currency }) => currency,
+    },
+    display: {
+      type: GraphQLString,
```

Comment:
I'm hesitant about this. Not the idea, but the implementation. I don't have a lot of confidence in `amount` in how it's implemented (which isn't a slight against whoever implemented it, it's just maintaining these things one off gives me pause). 

How I _think_  we should be really handling this is via [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) which was implemented by the ECMA standards body to be compliant with ISO standards...

The only challenge is, we'd actually have to build our own version of node to support this. See [this doc](https://nodejs.org/docs/latest-v10.x/api/intl.html#intl_internationalization_support) for more information.

Reply:
Reading further, it seems that we actually _wouldn't_ have to do a separate build. There are runtime options for adding this intl data. 

https://nodejs.org/docs/latest-v10.x/api/intl.html#intl_providing_icu_data_at_runtime

I think this is definitely worth looking into, but I feel like it's outside the scope of this PR. 

How would we best handle this? I can add noop arguments that we may want in the future to prevent usage here from being a breaking change.

Reply:
Oh thanks for pointing out `Intl.NumberFormat`! Yeah, I didn't suggest we implement it the same way but thought there might be need from the clients to format the price display differently. I'm 👍 to move forward for now and look into proper implementation separately. I'll make a note in the notion doc.

Is adding optional arguments a breaking change?

Reply:
Thanks. I think changing the arguments is breaking, yes, but emission isn't using display right now (nothing really is) so let's proceed and follow up.